velox/common/base/PeriodicStatsReporter.cpp

```diff
@@ -228,4 +232,13 @@ void PeriodicStatsReporter::reportCacheStats() {
   lastCacheStats_ = cacheStats;
 }
 
+void PeriodicStatsReporter::reportSpillStats() {
+  const auto spillMemoryStats = velox::memory::spillMemoryPool()->stats();
+  LOG(INFO) << "Spill memory usage: current["
```

Comment:
nit: consider to have VELOX_STATS_LOG with [STATS] prefix to ease log search. Thanks!

Reply:
I did not find VELOX_STATS_LOG